kmath-ast/src/jsTest/kotlin/space/kscience/kmath/estree/TestESTreeConsistencyWithInterpreter.kt

```diff
@@ -55,40 +54,44 @@ internal class TestESTreeConsistencyWithInterpreter {
         }
 
         assertEquals(
-            mst.interpret(ByteRing, Symbol.x to 3.toByte()),
-            mst.compile(ByteRing, Symbol.x to 3.toByte())
+            mst.interpret(ByteRing, x to 3.toByte()),
+            mst.compile(ByteRing, x to 3.toByte())
         )
     }
 
     @Test
-    fun realField() {
+    fun doubleField() {
         val mst = MstField {
             +(3 - 2 + 2 * number(1) + 1.0) + binaryOperationFunction("+")(
-                (3.0 - (bindSymbol("x") + (scale(add(number(1.0), number(1.0)), 2.0) + 1.0))) * 3 - 1.0
+                (3.0 - (bindSymbol(x) + (scale(add(number(1.0), number(1.0)), 2.0) + 1.0))) * 3 - 1.0
                         + number(1),
                 number(1) / 2 + number(2.0) * one
             ) + zero
         }
 
         assertEquals(
-            mst.interpret(DoubleField, Symbol.x to 2.0),
-            mst.compile(DoubleField, Symbol.x to 2.0)
+            mst.interpret(DoubleField, x to 2.0),
+            mst.compile(DoubleField, x to 2.0)
         )
     }
 
     @Test
     fun complexField() {
         val mst = MstField {
             +(3 - 2 + 2 * number(1) + 1.0) + binaryOperationFunction("+")(
-                (3.0 - (bindSymbol("x") + (scale(add(number(1.0), number(1.0)), 2.0) + 1.0))) * 3 - 1.0
+                (3.0 - (bindSymbol(x) + (scale(add(number(1.0), number(1.0)), 2.0) + 1.0))) * 3 - 1.0
```

Comment:
Why not use builders instead of those monstrosities

Reply:
Can be done in further patches. I'm that the test cases should be commonized and revised.